resources/assets/coffee/react/mp-history/main.coffee

```diff
@@ -64,7 +74,8 @@ class MPHistory.Main extends React.Component
         events: newEvents
         allEventsCount: data.all_events_count
         users: newUsers
-        since: _.last(newEvents).id
+        since: if !lastGameEvent? || lastGameEvent.game.data.end_time then lastEvent.id else lastGameEvent.id - 1
+        lastGameId: if lastGameEvent? then lastGameEvent.id else 0
         disbanded: lastEvent.event_type == 'match-disbanded'
```

Comment:
this can be removed and let first line of `@loadHistory` be 

```coffeescript
return if _.last(@state.events)?.detail.type == 'match-disbanded'
```
instead.

(and I guess the current one has some kind of timing problem because `setState` is async.)